tests/test_interconnect/Interconnect.json

```diff
@@ -0,0 +1,13043 @@
+{"top":"global.Interconnect",
```

Comment:
What's this? Please delete them. You might want to add `*.json` to the `.gitignore`

Reply:
sorry this slipped through the cracks - taking out